### DISCUSSION TITLE
it/hub/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#19009] [Docs]
# it/hub/

Scoprite Ultralytics HUB, lo strumento web all-in-one per l'addestramento e la distribuzione dei modelli YOLOv5 e YOLOv8 . Iniziate rapidamente con modelli pre-addestrati e funzioni di facile utilizzo.

https://docs.ultralytics.com/it/hub/

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for your interest in Ultralytics HUB 🚀! We appreciate you highlighting Ultralytics HUB, our all-in-one web tool for seamless training and deployment of YOLOv5 and YOLOv8 models. The HUB is packed with features designed to simplify your ML workflows and help you get started quickly with pre-trained models. Check out more details in our [Ultralytics HUB Documentation](https://docs.ultralytics.com/it/hub/) 📚.

If you have questions 🧐 or encounter any issues while using HUB, feel free to share more details here. If it's a 🐛 Bug Report, please include a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) whenever possible so we can assist you effectively.

💡 For guidance on YOLO usage, training tips, or troubleshooting solutions, our [Docs](https://docs.ultralytics.com/) offer a variety of helpful resources. You can also connect with our amazing community through [Discord](https://discord.com/invite/ultralytics), [Discourse](https://community.ultralytics.com/), or our [Subreddit](https://reddit.com/r/Ultralytics) for lively discussions and real-time support.

### Upgrade

Make sure you’re on the latest version of the `ultralytics` package by upgrading in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

### Explore Verified Environments

Ultralytics HUB and YOLO can be used in any of the following environments, which come pre-installed with all dependencies for smooth operation:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM ([Quickstart](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/))
- **Amazon** Deep Learning AMI ([Quickstart](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/))
- **Docker Image** ([Quickstart](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/)) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

### Current CI Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

All [Ultralytics CI workflows](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) are rigorously tested across macOS, Windows, and Ubuntu daily. A green badge indicates passing tests that confirm the proper functionality of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/).

🚀 This is an automated response, but no worries—an Ultralytics engineer will join the discussion soon to assist you further!
